Update phonenumbers extension

leeeennyy opened the pull request:
Added a new extension method which tries get the phonenumber formatted in E164 after validating it against the countries specified in order of preference.

leeeennyy committed:
- Add extension method for LibPhoneNumber which tries get the formatted phone number after validation in the countries specified
- Created test project and wrote a test for the new extension method

leeeennyy committed:
Merge branch 'master' into update-phonenumbers-extension

leeeennyy committed:
Update NUnit to version 3.6.1

leeeennyy committed:
Update the assembly version for LibPhoneNumber.Contrib since we added a new extension method.

leeeennyy committed:
Added TestFixture to the Test class

lukemcgregor commented on LibPhoneNumber.Contrib.Tests/PhoneNumberUtilExtensionsTests.cs:
```diff
@@ -0,0 +1,26 @@
+		[TestCase("0467751171", true, "+61467751171")]
+		public void TryGetFormattedPhoneNumberTest(string phoneNumber, bool tryGetResult, string outPhoneNumber)
+		{
+			var countryCodes = new [] { "NZ", "AU" };
```
I think put these into the test args maybe? and then test the reverse order

lukemcgregor commented on LibPhoneNumber.Contrib.Tests/Properties/AssemblyInfo.cs:
```diff
@@ -0,0 +1,36 @@
+// by using the '*' as shown below:
+// [assembly: AssemblyVersion("1.0.*")]
+[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyFileVersion("1.0.0.0")]
```
Do the 
```
[assembly: AssemblyVersion(AssemblyMeta.Version)]
[assembly: AssemblyFileVersion(AssemblyMeta.Version)]
[assembly: AssemblyInformationalVersion(AssemblyMeta.Version)]

internal static class AssemblyMeta
{
	public const string Version = "1.30.1";
}
```
thing

lukemcgregor reviewed (commented):
Couple of minor things you could fix if you want.
Would also be good to turn on XML comments for the project (all configurations) so the XML docs go into the nuget package

leeeennyy committed:
- Added XML docs to LibPhoneNumber project and added in missing comments
- Moved country code to be a testcase argument and added extra testcases with the country codes reversed

lukemcgregor reviewed (approved):
Looks great :+1:

leeeennyy committed:
- Added method to get formatted phonenumber given a phonenumber and country codes
